packages/lesswrong/server/apolloServer.ts

```diff
@@ -174,6 +174,7 @@ export function startWebserver() {
 
   app.use('/graphql', express.json({ limit: '50mb' }));
   app.use('/graphql', express.text({ type: 'application/graphql' }));
+  //perfMetricMiddleware needs to run after addAuthMiddlewares, so that it can access the user that passport puts on the request
```

Comment:
This is true of ~all the middlewares that happen after auth. I would add this to the `addAuthMiddlewares` line.